docs/audio-streamer.md

```diff
@@ -0,0 +1,170 @@
+# Stream audio
+
+The `AudioStreamer` class provides you with the ability to stream audio from a microphone in your .NET MAUI application through an eventhandler. The `AudioStreamer` works pretty much the same as the recorder `AudioRecorder`, except that it provides raw audio data instead of a file. In order to create an `AudioStreamer` instance you can make use of the `CreateStreamer` method on the [`AudioManager`](../readme.md#audiomanager) class.
```

Comment:
```suggestion
The `AudioStreamer` class provides you with the ability to stream audio from a microphone in your .NET MAUI application through an event handler. The `AudioStreamer` works pretty much the same as the recorder `AudioRecorder`, except that it provides raw audio data instead of a file. In order to create an `AudioStreamer` instance you can make use of the `CreateStreamer` method on the [`AudioManager`](../readme.md#audiomanager) class.
```